sematic/ui/src/index.tsx

```diff
@@ -183,6 +185,34 @@ function Router() {
   return <RouterProvider router={router} />;
 }
 
+posthog.init( // new
+    'phc_nJlFf7MpsrzF5pPaSEQi5GKyTSDjHRcqqL808VLRNXc', { 
```

Comment:
Maybe we should wait for the acknowledgment that including the key in the source is OK.